usaspending_api/etl/elasticsearch_loader_helpers/es_etl_helpers.py

```diff
@@ -558,12 +610,22 @@ def post_to_elasticsearch(client, job, config, chunksize=250000):
             logger.info(format_log(f"No documents to add/delete for chunk #{count}", job=job.name, process="ES Index"))
             continue
 
+        # Only delete before adding/inserting/indexing new docs on incremental loads, not full reindexes
+        is_incremental = config["is_incremental_load"] and str(config["is_incremental_load"]).lower() == "true"
```

Comment:
note to self: refactor this as it seems like Javascript. `config["is_incremental_load"]` is already set as a bool

Reply:
So I went back and forth on this. If some string data got injected into that dict key, like `"false"`, it would still eval as truthy, yes?

Reply:
`config["is_incremental_load"]` is not user-provided. It is calculated based on other configuration values and set with a boolean value.

```python
    config["is_incremental_load"] = not bool(config["create_new_index"]) and (
         config["starting_date"] != default_datetime
)
```